dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -947,4 +949,33 @@ private String getPermissionFilterQuery(DataverseRequest dataverseRequest, SolrQ
 
     }
 
+    public String getLocaleFacetName(String name, List<DatasetFieldType> datasetFields ) {
```

Comment:
I would add overloaded method that doesn't take datasetField argument, plus I would add javadoc that would explain the context of this method (why we searching through few bundles).